### PR TITLE
Add LICENSE and CHANGELOG

### PR DESCRIPTION
**Pull request by itskingori**
Title says it all.

### CODE REVIEW

**Comment by itskingori** [comment]
WIP

**Comment by itskingori** [comment]
No longer WIP.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Intellection Software Ltd
```
Comment (adrianmoisey) [inline LICENSE]: 2016 should be 2017.

And I'm not 100% sure on the `Intellection Software Ltd`

**Comment by adrianmoisey** [comment]
While you're here, can you change the badge in the README to have `style=shield` instead of `style=svg`?

**Comment by adrianmoisey** [comment]
Besides the minor comments, :+1:

**Review by adrianmoisey — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (itskingori) [reply LICENSE]: Good catch. Fixed in 48e09c1.

**Comment by itskingori** [comment]
> While you're here, can you change the badge in the README to have `style=shield` instead of `style=svg`?

Done in 1cea1ce.

**Pull request merged**